dir.props

```diff
@@ -26,6 +26,11 @@
       <Value>True</Value>
     </AssemblyMetadata>
   </ItemGroup>
+  
+  <!-- Temporary adding this property so that Linux builds will target the net45 version of the task library instead of the .Net Core version -->
```

Comment:
You say "Linux". What about OS X, FreeBSD, etc.?

I also don't understand what it means to for these UNIX build to target something other than .NET Core. Can you elaborate?